.github/workflows/tb_controller_integration_test.yaml

```
@@ -0,0 +1,64 @@
+name: Tensorboard Controller Integration Test
+on:
+  pull_request:
+    paths:
+      - components/tensorboard-controller/**
+      - releasing/version/VERSION
+    branches:
+      - master
+      - v*-branch
+      - notebooks-v1
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event.number }}
+  cancel-in-progress: true
+
+env:
+  IMG: ghcr.io/kubeflow/kubeflow/tensorboard-controller
```

```suggestion
  IMG: ghcr.io/kubeflow/notebooks/tensorboard-controller
```

As we are migrating the workflows to this `kubeflow/notebooks` repository - we need to make sure the `IMG` reference is suitably _aligned_